.github/workflows/release.yml

```diff
@@ -211,39 +164,53 @@ jobs:
           java-version: 8
           cache: sbt
 
-      - name: Make _install/lib directory to store lib files
-        run: mkdir -p _install/lib
-
-      - name: Download linux library file 🔻
-        uses: actions/download-artifact@v3
-        with:
-          name: libomega_edit.so
-          path: _install/lib/libomega_edit.so
-
-      - name: Move out library file 🛻
-        run: |
-          LIB_FILENAME="libomega_edit.so"
-          mv -v "_install/lib/${LIB_FILENAME}" "_install/lib/${LIB_FILENAME}_dir"
-          mv -v "_install/lib/${LIB_FILENAME}_dir/$LIB_FILENAME" "_install/lib/$LIB_FILENAME"
-          rm -rf "_install/lib/${LIB_FILENAME}_dir"
-        shell: bash
+      # - name: Make _install/lib directory to store lib files
+      #   run: mkdir -p _install/lib
+
+      # - name: Download linux library file 🔻
+      #   uses: dawidd6/action-download-artifact@v2
+      #   with:
+      #     workflow: tests.yml
+      #     workflow_conclusion: success
+      #     name: libomega_edit.so
+      #     path: _install/lib/libomega_edit.so
+
+      # - name: Move out library file 🛻
+      #   run: |
+      #     LIB_FILENAME="libomega_edit.so"
+      #     mv -v "_install/lib/${LIB_FILENAME}" "_install/lib/${LIB_FILENAME}_dir"
+      #     mv -v "_install/lib/${LIB_FILENAME}_dir/$LIB_FILENAME" "_install/lib/$LIB_FILENAME"
+      #     rm -rf "_install/lib/${LIB_FILENAME}_dir"
+      #   shell: bash
```

Comment:
Remove commented out code.  If we need it, it's in the repo.

Reply:
This not the same code that is already in the repo. The download of the library file downloads from a different workflow. I want to keep it in till I make sure the current process I have works, because if it doesn't this code will need uncommented and used.

Reply:
If it does work, then this will need to be removed then.  You can always get the code back from git.

Reply:
It would be removed the updated commit for node_publish of 0.9.66 if the dry run works out properly